R/sim_linelist.R

```diff
@@ -126,6 +126,18 @@
 #' onset-to-recovery times in the line list may not resemble the distributional
 #' form input into the function.
 #'
+#' @param reporting_delay A `function` for the reporting delay distribution or
+#' `NULL`. The (random) number generating function creates delays between the
+#' time of symptom onset (`$date_onset`) and the case being reported
+#' (`$date_reporting`).
+#'
+#' The function can be defined or anonymous. The function must return a vector
+#' of `numeric`s for the length of the reporting delay. The function must have
```

Comment:
I find this a bit difficult to parse - perhaps say that the length of the vector is the number of random numbers to generate?